dev/com.ibm.ws.logging/src/com/ibm/ws/logging/collector/CollectorJsonUtils.java

```diff
@@ -230,16 +230,18 @@ else if (key.equals(LogFieldConstants.MESSAGE)) {
                 kvpl = (KeyValuePairList) p;
                 if (kvpl.getName().equals(LogFieldConstants.EXTENSIONS_KVPL)) {
                     extensions = kvpl.getKeyValuePairs();
-                    boolean isExtQuoteless = false;
                     for (KeyValuePair k : extensions) {
                         String extKey = k.getKey();
-                        String extValue = k.getStringValue();
-                        // Checking the value of the extension KeyValuePair to make sure the suffix is correct
-                        boolean isValidExt = CollectorJsonHelpers.checkExtSuffixValidity(k);
-                        if (isValidExt) {
-                            isExtQuoteless = CollectorJsonHelpers.checkIfExtIsQuoteless(extKey);
-                            extKey = LogFieldConstants.EXT_PREFIX + extKey;
-                            CollectorJsonHelpers.addToJSON(sb, extKey, extValue, false, true, false, false, isExtQuoteless);
+                        if (extKey.endsWith(CollectorJsonHelpers.INT_SUFFIX)) {
+                            CollectorJsonHelpers.addToJSON(sb, extKey, k.getIntValue().toString(), false, true, false, false, k.isInteger());
```

Comment:
last parameter, k.isInteger(), should be replaced with true.  (and similarly for isFloat, isBoolean, isLong below)